Lottie-Windows-WinUI3/Lottie-Windows-WinUI3.csproj

```
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Library</OutputType>
+		<RootNamespace>MicrosoftToolkit.WinUI.Lottie</RootNamespace>
```

```suggestion
		<RootNamespace>CommunityToolkit.WinUI.Lottie</RootNamespace>
```

`MicrosoftToolkit` is not our root namespace.